### DISCUSSION TITLE
Where is the favicon code?

### DISCUSSION THREAD

**Discussion by Benhrpr** [alshedivat/al-folio#1607] [Q&A]
I am trying to find the code that utilises an emoji as the favicon, any ideas?

**Comment by george-gca** [comment]
You can simply copy an emoji from [emojipedia](https://emojipedia.org/) and paste it in your [_config.yml](https://github.com/alshedivat/al-folio/blob/c824d128673c000ef9f0e0e77a0ae30d2c4b4568/_config.yml#L19).

**Reply by Benhrpr** [reply]
Thanks but I wanted to know how to do it for a different project.

**Reply by george-gca** [reply]
Everything that is defined in `_config.yml` is referenced as `site.XXX`, so in this case it is referenced as `site.icon`. It is used in the [_includes/head.html](https://github.com/alshedivat/al-folio/blob/c824d128673c000ef9f0e0e77a0ae30d2c4b4568/_includes/head.html#L25).
